src/modules/components/proposal/proposalVoting/proposalVotingContainer/proposalVotingContainer.tsx

```diff
@@ -1,17 +1,26 @@
-import classNames from 'classnames';
-import React, { Children, type ComponentProps } from 'react';
-import { Accordion, Card, Heading } from '../../../../../core';
-import { type IProposalVotingStageProps } from '../proposalVotingStage';
+import { Children, cloneElement, isValidElement, type ComponentProps, type ReactElement } from 'react';
+import { Accordion } from '../../../../../core';
 
-export interface IProposalVotingContainerProps extends ComponentProps<'div'> {
+interface ChildProps {
     /**
-     * Title displayed on top.
+     * Index of the child in the parent.
      */
-    title: string;
+    index: number;
     /**
-     * Description of the proposal voting.
+     * Active stage that will be expanded for multi-stage proposals.
+     */
+    activeStage?: string;
+    /**
+     * Callback called when the user selects a stage, to be used for expanding the current active stage for multi-stage proposals.
      */
-    description: string;
+    onStageClick?: (stage?: string) => void;
```

Comment:
Why these changes? Can't we use the `IProposalVotingStageProps` as we did before?